backend/core/src/seeder/seeds/listings/shared.ts

```diff
@@ -242,7 +242,21 @@ export const liveWorkPreference: MultiselectQuestionSeedType = {
   ],
   optOutText: "I don't want this preference",
   options: [
-    { text: "Live in County", ordinal: 1 },
+    {
+      text: "Live in County",
+      ordinal: 1,
+      description: "A description of the option.",
+      links: [
+        {
+          title: "Link Title",
+          url: "https://www.example.com",
+        },
+        {
+          title: "Link Title",
+          url: "https://www.example.com",
+        },
+      ],
```

Comment:
Can you update the title of one of these, it made me think there was a bug when reviewing the frontend.